README.md

```diff
@@ -1,229 +1,60 @@
 
-```Makefile
-## Image tools  (https://github.com/scaleway/image-tools)
-all:    docker-rules.mk
-docker-rules.mk:
-    wget -qO - https://j.mp/scw-builder | bash
--include docker-rules.mk
 ```
-
-
-# Unit test a running instance
-
-At runtime, you can proceed to unit tests by calling
-
-```bash
-# using curl
-SCRIPT=$(mktemp); curl -s -o ${SCRIPT} https://raw.githubusercontent.com/scaleway/image-tools/master/builder/unit.bash && bash ${SCRIPT}
-# using wget
-SCRIPT=$(mktemp); wget -qO ${SCRIPT} https://raw.githubusercontent.com/scaleway/image-tools/master/builder/unit.bash && bash ${SCRIPT}
+# Using the scw tool -- this assumes you've already set it up.
+# Create a snapshot first:
+$ scw commit -v 1 my-server my-snapshot
+f3af311c-53f7-4f5d-9252-0bf69f017269
+
+# Now make an image out of it:
+$ scw tag --arch=x86_64 --bootscript="mainline 4.14"  my-snapshot my-image
+a601dbac-08cb-4af3-9ff6-73b7e8dfd34f
+
+$ scw images
+REPOSITORY           TAG                 IMAGE ID            CREATED             REGION              ARCH
+user/my-image        latest              a601dbac            5 seconds           [     par1]         [x86_64]
+---- snipping all stock images 8< ----
 ```
 
-# Image check list
-
-List of features, scripts and modifications to check for proper **scaleway** image creation.
-
-- [ ] Add sysctl entry `vm.min_free_kbytes=65536`
-- [ ] Configure NTP to use internal server
-- [ ] Configure SSH to only accept login through public keys and deny environment customization to avoid errors due to users locale
-- [ ] Configure default locale to `en_US.UTF-8`
-- [ ] Configure network scripts to use DHCP and enable them
-  Although not, strictly speaking, needed since kernel already has IP address and gateway this allows DHCP hooks to be called for setting hostname, etc
-- [ ] Install custom DHCP hook for hostname to set entry in `/etc/hosts` for software using `getent_r` to get hostname
-- [ ] Install scripts to fetch SSH keys
-- [ ] Install scripts to fetch kernel modules
-- [ ] Install scripts to connect and/or mount NBD volumes
-- [ ] Install scripts to manage NBD root volume
-- [ ] Disable all physical TTY initialization
-- [ ] Enable STTY @ 9600 bps
-
-Before making the image public, do not forget to check it boots, stops and restarts from the OS without any error (most notably kernel) since a failure could lead to deadlocked instances.
+### Docker-based images
 
+Using the tools contained in this repository, it is possible to create Scaleway images from Docker images. At its core, this process uses the tools we explored above and is pretty straightforward to use: 
+  1. Clone this repository and the repository of the image (if there is one) on a Scaleway instance
+  2. In your terminal, navigate to the image-tools directory, and type `make IMAGE_DIR=<the image's directory> scaleway_image`
+  3. ???
```

Comment:
What kind of push are we talking about ? A Dockerhub push ?

Reply:
It's an example it could be a "ScalewayHub" push. The key part is having a way for users to keep an artifact home in case they want to recreate a new account with the same images.